src/main/resources/controllers-context.xml

```diff
@@ -31,6 +31,20 @@
         <property name = "password" value = "${db.password}"/>
     </bean>
 
+    <bean id="flyway" class="org.flywaydb.core.Flyway" init-method="migrate">
+        <constructor-arg ref="flywayConfig"/>
+    </bean>
+
+    <bean id = "flywayConfig" class = "org.flywaydb.core.api.configuration.ClassicConfiguration">
```

Comment:
You don't need to expose this bean. You'll inject it only in 1 place. So it's better to make it a nested bean to the original "flyway" bean.